Update enable-an-exporting-connector.md (#17943)
netdata/netdata: docs/exporting-metrics/enable-an-exporting-connector.md

connector itself. We'll walk through the process of enabling the exporting engine itself, followed by two examples using
the OpenTSDB and Graphite connectors.

> **Note**
>
> When you enable the exporting engine and a connector, the Netdata Agent exports metrics _beginning from the time you
> restart its process_, not the entire
> [database of long-term metrics](/docs/netdata-agent/configuration/optimizing-metrics-database/change-metrics-storage.md).

> Once you understand the process of enabling a connector, you can carry that knowledge to any other connector.

Reviewer: ```suggestion
Once you understand how to enable a connector, you can apply that knowledge to any other connector.
```

    destination = localhost:4242
```

Replace `my_opentsdb_http_instance` with an instance name of your choice, and change the `destination` setting to the IP address or hostname of your OpenTSDB database.

> [Restart your Agent](/docs/netdata-agent/start-stop-restart.md), to begin exporting to your OpenTSDB database. The Netdata Agent exports metrics _beginning from the time the process starts_, and because it exports as metrics are collected, you should start seeing data in your external database after only a few seconds.

Reviewer: ```suggestion
[Restart your Agent](/docs/netdata-agent/start-stop-restart.md) to initiate exporting to your OpenTSDB database. The Netdata Agent continuously exports metrics collected from the moment it starts. You can expect to see data appear in your OpenTSDB database within seconds of restarting the Agent.
```
